### ISSUE TITLE
Documentation edits in `split.R` Examples

### ISSUE THREAD

**Issue by feinleib** [tidyverse/stringr#519]
This is related to issue #500. I initially commented there, but I'm creating a new issue because PR #502 already fixed that issue.

I have two more fixes in the Examples section of `split.R` documentation:

1. The comment for `str_split_1()` is missing the second underscore.
```
# If you want to split a single string, use `str_split1`
```

2. The names of the functions are written differently in each comment. These should be made consistent with each other.
```
# If you want to split a single string, use `str_split1`
# Use fixed to return a character matrix
# str_split_i extracts only a single piece from a string
```

My understanding of the [relevant section of the Tidyverse Style Guide](https://style.tidyverse.org/documentation.html#r-code) is that the correct format for these comments is backticked, with trailing parentheses:
```
`str_split_1()`     `str_split_fixed()`     `str_split_i()`
```

**Comment by hadley** [comment]
Would you like to do a PR to fix this

**Comment by hadley** [comment]
Fixed by #526
